src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQueryAvroToStructuredTransformer.java

```diff
@@ -86,8 +91,15 @@ protected Object convertField(Object field, Schema fieldSchema) throws IOExcepti
           case TIMESTAMP_MICROS:
             return field;
           case DATETIME:
-            //BQ datetime should already be in the expected ISO 8601 format for LocalDateTime
-            return LocalDateTime.parse(field.toString());
+            try {
+              return LocalDateTime.parse(field.toString());
```

Comment:
think it's a little better to parse for validation, but still return the string. That way we don't need the instanceof check above.